packages/commonwealth/client/scripts/views/components/sidebar/CommunitySection/ProfileCard.tsx

```diff
@@ -0,0 +1,52 @@
+import { useFetchProfileByIdQuery } from 'client/scripts/state/api/profiles';
+import useUserStore from 'client/scripts/state/ui/user';
+import clsx from 'clsx';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './ProfileCard.scss';
+
+const ProfileCard = () => {
+  const userData = useUserStore();
+
+  const { data } = useFetchProfileByIdQuery({
+    apiCallEnabled: userData.isLoggedIn,
+  });
+
+  const backgroundImageUrl = data?.profile?.background_image?.url || '';
+  const backgroundBehavior =
+    data?.profile?.background_image?.imageBehavior || 'cover';
+
+  return (
+    <div className="profile-card">
```

Comment:
```suggestion
    <div className="ProfileCard">
```
please remember that in other PRs